Enable jumbo build for the following component(s)/source_set(s):

- //device/fido:fido
- //device/gamepad:gamepad

diff -uNr a/device/fido/BUILD.gn b/device/fido/BUILD.gn
--- a/device/fido/BUILD.gn
+++ b/device/fido/BUILD.gn
@@ -4,9 +4,10 @@
 
 import("//build/config/chromeos/ui_mode.gni")
 import("//build/config/features.gni")
+import("//build/config/jumbo.gni")
 import("//testing/libfuzzer/fuzzer_test.gni")
 
-component("fido") {
+jumbo_component("fido") {
   sources = [
     "attestation_object.cc",
     "attestation_object.h",
diff -uNr a/device/fido/aoa/android_accessory_device.cc b/device/fido/aoa/android_accessory_device.cc
--- a/device/fido/aoa/android_accessory_device.cc
+++ b/device/fido/aoa/android_accessory_device.cc
@@ -14,6 +14,9 @@
 #include "components/device_event_log/device_event_log.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
+#define kTimeoutMilliseconds     kTimeoutMilliseconds_AADE
+#define kLongTimeoutMilliseconds kLongTimeoutMilliseconds_AADE
+
 namespace device {
 
 static constexpr unsigned kTimeoutMilliseconds = 1000;
@@ -152,3 +155,6 @@
 }
 
 }  // namespace device
+
+#undef kTimeoutMilliseconds
+#undef kLongTimeoutMilliseconds
diff -uNr a/device/fido/aoa/android_accessory_discovery.cc b/device/fido/aoa/android_accessory_discovery.cc
--- a/device/fido/aoa/android_accessory_discovery.cc
+++ b/device/fido/aoa/android_accessory_discovery.cc
@@ -20,6 +20,9 @@
 
 #include "base/functional/bind.h"
 
+#define kTimeoutMilliseconds     kTimeoutMilliseconds_AAD
+#define kLongTimeoutMilliseconds kLongTimeoutMilliseconds_AAD
+
 // See https://source.android.com/devices/accessories/aoa for details on the
 // protocol used to talk to apps on the phone here.
 
@@ -495,3 +498,6 @@
 }
 
 }  // namespace device
+
+#undef kTimeoutMilliseconds
+#undef kLongTimeoutMilliseconds
diff -uNr a/device/fido/get_assertion_request_handler.cc b/device/fido/get_assertion_request_handler.cc
--- a/device/fido/get_assertion_request_handler.cc
+++ b/device/fido/get_assertion_request_handler.cc
@@ -48,6 +48,8 @@
 #include "device/fido/cros/authenticator.h"
 #endif
 
+#define AtLeastUVPreferred AtLeastUVPreferred_GARH
+
 namespace device {
 
 namespace {
@@ -823,3 +825,5 @@
 }
 
 }  // namespace device
+
+#undef AtLeastUVPreferred
diff -uNr a/device/fido/make_credential_request_handler.cc b/device/fido/make_credential_request_handler.cc
--- a/device/fido/make_credential_request_handler.cc
+++ b/device/fido/make_credential_request_handler.cc
@@ -39,6 +39,8 @@
 #include "device/fido/cros/authenticator.h"
 #endif
 
+#define AtLeastUVPreferred AtLeastUVPreferred_MCRH
+
 namespace device {
 
 using PINUVDisposition = FidoAuthenticator::PINUVDisposition;
@@ -1090,3 +1092,5 @@
 }
 
 }  // namespace device
+
+#undef AtLeastUVPreferred
diff -uNr a/device/gamepad/BUILD.gn b/device/gamepad/BUILD.gn
--- a/device/gamepad/BUILD.gn
+++ b/device/gamepad/BUILD.gn
@@ -4,6 +4,7 @@
 
 import("//build/config/chromeos/ui_mode.gni")
 import("//build/config/features.gni")
+import("//build/config/jumbo.gni")
 
 if (is_android) {
   import("//build/config/android/config.gni")
@@ -11,7 +12,7 @@
   import("//third_party/jni_zero/jni_zero.gni")
 }
 
-component("gamepad") {
+jumbo_component("gamepad") {
   output_name = "device_gamepad"
 
   sources = [
diff -uNr a/device/gamepad/dualshock4_controller.cc b/device/gamepad/dualshock4_controller.cc
--- a/device/gamepad/dualshock4_controller.cc
+++ b/device/gamepad/dualshock4_controller.cc
@@ -15,6 +15,9 @@
 #include "device/gamepad/hid_writer.h"
 #include "device/gamepad/public/cpp/gamepad_features.h"
 
+#define kRumbleMagnitudeMax kRumbleMagnitudeMax_DualShock4
+#define ControllerData      ControllerData_DualShock4
+
 namespace device {
 
 namespace {
@@ -404,3 +407,6 @@
 }
 
 }  // namespace device
+
+#undef kRumbleMagnitudeMax
+#undef ControllerData
diff -uNr a/device/gamepad/nintendo_controller.cc b/device/gamepad/nintendo_controller.cc
--- a/device/gamepad/nintendo_controller.cc
+++ b/device/gamepad/nintendo_controller.cc
@@ -13,6 +13,8 @@
 #include "device/gamepad/gamepad_data_fetcher.h"
 #include "device/gamepad/gamepad_id_list.h"
 
+#define ControllerData ControllerData_Nintendo
+
 namespace device {
 namespace {
 // Device IDs for the Switch Charging Grip, also used for composite devices.
@@ -1773,3 +1775,5 @@
 }
 
 }  // namespace device
+
+#undef ControllerData
diff -uNr a/device/gamepad/xbox_hid_controller.cc b/device/gamepad/xbox_hid_controller.cc
--- a/device/gamepad/xbox_hid_controller.cc
+++ b/device/gamepad/xbox_hid_controller.cc
@@ -7,6 +7,8 @@
 #include "device/gamepad/gamepad_id_list.h"
 #include "device/gamepad/hid_writer.h"
 
+#define kRumbleMagnitudeMax kRumbleMagnitudeMax_XboxHid
+
 namespace device {
 
 namespace {
@@ -59,3 +61,5 @@
 }
 
 }  // namespace device
+
+#undef kRumbleMagnitudeMax
